docs/advanced/lanes.md

```diff
@@ -212,7 +196,24 @@ lane_context[SharedValues::GRADLE_FLAVOR]               # Generated by `gradle`
 lane_context[SharedValues::GRADLE_BUILD_TYPE]           # Generated by `gradle`
 ```
 
-To get information about available lane variables, run `fastlane action [action_name]`.
+To get information about available lane variables, run `fastlane action [action_name]` or look at the generated table in the action documentation.
+
+# Lane Properties
```

Comment:
Have to know about `lane_context` first before being able to introduce the properties.